lib/archive_zip_manager.rb

```diff
@@ -50,8 +50,11 @@ def add_bank_transactions(zipfile)
 
   def add_expenses(zipfile)
     zipfile.dir.mkdir("expenses")
+    zipfile.dir.chdir("expenses")
     expenses.each do |expense|
-      add_file_to_archive(zipfile, 'expenses', expense.attachment) if expense.attachment
+      user_folder = "#{expense.user.first_name}#{expense.user.last_name}".parameterize
+      zipfile.dir.mkdir(user_folder) rescue nil
```

Comment:
Is the `rescue nil` here to rescue when the directory already exist?
If so, maybe better use `mkdir_p`, if it works...